frontends/precinct-scanner/src/app_root.tsx

```diff
@@ -370,21 +368,26 @@ export function AppRoot({
     return <SetupPowerPage />;
   }
 
+  if (auth.status === 'checking_passcode' && auth.user.role === 'superadmin') {
+    return <UnlockMachineScreen auth={auth} />;
+  }
```

Comment:
This was an interesting find - prior to this PR, when VxScan was unconfigured (i.e. without an election definition), inserting a sys admin card didn't do anything. This check now makes it such that one can always access the sys admin screen. I add a test case for this later in this PR